src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -54,13 +80,33 @@ parseObjectID(
 }
 
 static Expected<uint256, Json::Value>
-parseIndex(Json::Value const& params, Json::StaticString const fieldName)
-{
+parseIndex(
+    Json::Value const& params,
+    Json::StaticString const fieldName,
+    unsigned apiVersion)
+{
+    if (apiVersion > 2u && params.isString())
```

Comment:
Do we have a catalog of what is going into APIv3 yet? I don't want this to fall through the cracks when it's time to get feedback on/release APIv3.

Reply:
cc @bthomee